rules/S7495/python/metadata.json

```
@@ -0,0 +1,23 @@
+{
+  "title": "Direct dictionaries and sets comprehensions should be preferred over list comprehensions when creating dictionaries and sets respectively",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "5min"
+  },
+  "tags": [
+  ],
+  "defaultSeverity": "Major",
+  "ruleSpecification": "RSPEC-7495",
+  "sqKey": "S7495",
+  "scope": "All",
+  "defaultQualityProfiles": ["Sonar way"],
+  "quickfix": "unknown",
```

Shouldn't a quickfix be possible for this?
```suggestion
  "quickfix": "targeted",
```